### DISCUSSION TITLE
ArgoCD upgrade fron 2.2.5 to 2.4.3

### DISCUSSION THREAD

**Discussion by Mais316** [argoproj/argo-cd#9864] [Q&A] [answered]
Please advise, we have ArgoCD 2.2.5 and we need to upgrade to latest. Is it Ok to go to 2.4.3 directly or we have to go from 2.2.x to 2.3.x then to 2.4.x.

**Comment by wtam2018** [comment, marked as answer]
You can upgrade directly to the latest 2.4 without going to 2.3.x first. However, please review the incremental changes https://github.com/argoproj/argo-cd/tree/master/docs/operator-manual/upgrading that may affect you.
